gettingstarted.md

```diff
@@ -0,0 +1,132 @@
+# Getting started
+
+This briefly describe one way to getting started using easy-dotnet on the windows platform
+
+## Installing Neovim + extras
+
+We use chocolatey as the package manager. To get up and running we need a few tools on top of neovim itself. a C-compiler and ripgrep. Also .net of course
+
+```
+> choco install neovim
+> choco install ripgrep
+> choco install zip
+> choco install choco install dotnet-8.0-sdk
+> choco install git
+> choco install fzf
+```
+
+## Starting neovim
+you should now open a new terminal and start neovim
+
+```
+> nvim 
+```
+
+
+## Installing neovim plugin manager
+
+first we need to create the folder to the config file (we use powershell). Then we start neovim to configure
+
+``` 
+> 
+> mkdir $env:LOCALAPPDATA\nvim\
+> nvim $env:LOCALAPPDATA\nvim\init.lua
+```
+
+## Seting up neovim 
+
+Starting neovim is an underwhelming surprise the first time. Let's change that.
+
+add the following to `init.lua`
+
+```
+-- Set <space> as the leader key
+--  NOTE: Must happen before plugins are loaded (otherwise wrong leader will be used)vim.g.mapleader = ' '
+vim.g.maplocalleader = ' '
+
+-- Set to true if you have a Nerd Font installed and selected in the terminal
+vim.g.have_nerd_font = false
+
+vim.opt.number = true
+
+-- Enable mouse mode, can be useful for resizing splits for example!
+vim.opt.mouse = 'a'
+
+vim.opt.showmode = true
+
+-- add this if you want to be able to copy from neovim to windows programs
+vim.schedule(function() vim.opt.clipboard = 'unnamedplus' end)
```

Comment:
You can simply do, no need for vim.schedule
```lua
vim.opt.clipboard = "unnamedplus"
```

Reply:
it is taken from kickstart.nvim so I thought it was the recommended way

Reply:
Strange, it probably solves an edgecase I havent ran into. Probably just keep it then

Reply:
Faster startup